server/doc/swdesign/README.md

```diff
@@ -218,6 +218,133 @@ Needs:
 - impl
 - utest
 
+#### Server supports PEM file paths as cli arguments for mTLS
+`swdd~server-supports-pem-file-paths-as-cli-arguments~1`
+
+Status: approved
+
+The Ankaios server shall support the following cli arguments all of which contain a file path to a PEM file:
+* `--ca_pem`: file path to the root certificate authority
+* `--crt_pem`: file path to the certificate
+* `--key_pem`: file path to the key
+
+Comment:
+The PEM file format is a text-based, human-readable format. PEM files can contain certificates, private keys, public keys and even certificate chains, making them more versatile compared to other formats.
+
+Rationale:
+The Ankaios server uses the PEM files to establish a mutual transport layer security (mTLS) communication.
+
+Tags:
+- AnkaiosServer
+
+Needs:
+- impl
+- stest
+
+#### Server supports cli argument for insecure communication
+`swdd~server-supports-cli-argument-for-insecure-communication~1`
+
+Status: approved
+
+The Ankaios server shall support the cli argument `--insecure` with short alias `-k`.
+
+Comment:
+The Ankaios server supports an insecure communication channel to the Ankaios server. Communicating over insecure channels is convenient during an evaluation or a development phase.
+
+Tags:
+- AnkaiosServer
+
+Needs:
+- impl
+- stest
+
+#### Server establishes insecure connection on provided insecure cli argument
+`swdd~server-establishes-insecure-communication-based-on-provided-insecure-cli-argument~1`
+
+Status: approved
+
+When the user starts the Ankaios server with the arguments `-k` or `--insecure` and the Ankaios server initializes the communication middleware, then the Ankaios server shall instruct the communication middleware to use an insecure communication channel.
+
+Tags:
+- AnkaiosServer
+- CommunicationMiddleware
```

Comment:
We don't need to tag the CommunicationMiddleware. It is an external component and tagging it does not bring any benefit.
```suggestion
```